### PR TITLE
Bump: library-go to get verify-deps for go mod

### PR DESCRIPTION
**Pull request by tnozicka**
/cc @mfojtik

### CODE REVIEW

File: vendor/github.com/openshift/library-go/alpha-build-machinery/make/targets/openshift/deps-gomod.mk [hunk 1]
```diff
@@ -0,0 +1,41 @@
+self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
```
Comment (tnozicka) [inline vendor/github.com/openshift/library-go/alpha-build-machinery/make/targets/openshift/deps-gomod.mk]: here it goes

File: go.mod [hunk 2]
```diff
@@ -5,12 +5,11 @@ go 1.13
...
 	github.com/hashicorp/golang-lru v0.5.1
 	github.com/openshift/api v3.9.1-0.20190923175213-ecbc01928def+incompatible
-	github.com/openshift/client-go v0.0.0-20190923180330-3b6373338c9b
-	github.com/openshift/library-go v0.0.0-20190924092619-a8c1174d4ee7
+	github.com/openshift/client-go v0.0.0-20191216194936-57f413491e9e
+	github.com/openshift/library-go v0.0.0-20191218095328-1c12909e5923
 	go.uber.org/atomic v1.3.3-0.20181018215023-8dc6146f7569 // indirect
 	go.uber.org/multierr v1.1.1-0.20180122172545-ddea229ff1df // indirect
 	k8s.io/api v0.17.0
-	k8s.io/apiextensions-apiserver v0.17.0 // indirect
```
Comment (mfojtik) [inline go.mod]: wtf

Comment (tnozicka) [reply go.mod]: go mod tidy

Comment (tnozicka) [reply go.mod]: no direct use I can see.

steps I run:
```
go get -mod='' github.com/openshift/library-go@master
go mod tidy
go mod vendor
```

**Pull request merged**